agent/tests/input_files/raw/pipelines/test_pipeline.json

```diff
@@ -0,0 +1,43 @@
+[
```

Comment:
`input_files/raw/` is not the best place for this pipeline and its source, this folder contains configs for a specific kind of pipeline that saves data locally without transformations. Create a separate dir for them